installing/installing_sno/install-sno-installing-sno.adoc

```diff
@@ -105,4 +105,31 @@ You can use dedicated or shared IFLs to assign sufficient compute resources. Res
 
 include::modules/install-sno-ibm-z.adoc[leveloffset=+2]
 
-include::modules/install-sno-ibm-z-kvm.adoc[leveloffset=+2]
\ No newline at end of file
+include::modules/install-sno-ibm-z-kvm.adoc[leveloffset=+2]
+
+== Installing {sno} with {ibmpowerProductName}
+
+Installing a single node cluster on {ibmpowerProductName} requires user-provisioned installation using either the "Installing a cluster with {op-system-base} KVM on {ibmpowerProductName}" procedure.
```

Comment:
s/single node cluster/{sno} cluster ??

Reply:
I think it is correct. Referred to: https://github.com/openshift/openshift-docs/pull/63974/files

Reply:
We took it the wording from the AWS entry.